app/controllers/api/answers_controller.rb

```diff
@@ -18,6 +18,15 @@ def update
     end
   end
 
+  ##
+  # Validates that the email is valid address, then sets the status of the
+  # answer to be predelegated
+  #
+  def predelegate_answer(email)
+    answer = Answer.find(params[:id])
+    answer.set_status_predelegated(email) if email =~ /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
```

Comment:
this might be better as a custom validation method in the answer model. So if status = pre delegated, text should be valid email.
validate answer.text, using custom validation, on update.